afogel/PairApp
# 34 implement initial styling of app

# What changed?
This  PR converts the frontend to use daisy instead of bootstrap for existing landing, flash, navbar, and devise views.

## To test
Check out this branch, bundle and yarn install, then boot up your server with `bin/dev`. Stuff should look like the screenshots. You should also be able to click on the x in a flash message and dismiss it.

Ignore the purple "hello I'm a daisy button", that's only test copy on my machine. It will not show up in your version.

## Screenshots:
![Screen Shot 2023-05-11 at 5 01 22 PM](https://github.com/afogel/PairApp/assets/2447409/42926242-d111-42d1-a897-8cf0c75e063f)

![Screen Shot 2023-05-11 at 5 01 36 PM](https://github.com/afogel/PairApp/assets/2447409/18862ce5-8d5b-4147-9627-be4b7fdd1666)
![Screen Shot 2023-05-11 at 5 01 47 PM](https://github.com/afogel/PairApp/assets/2447409/e63cdb58-93f0-4547-8cf2-0558b8128587)
![Screen Shot 2023-05-11 at 5 02 11 PM](https://github.com/afogel/PairApp/assets/2447409/a86b10e0-d25f-4ca3-998e-ed2951b2eb68)
![Screen Shot 2023-05-11 at 5 02 22 PM](https://github.com/afogel/PairApp/assets/2447409/3f419885-8ecd-43b8-b267-f1ac2c25fa1f)
![Screen Shot 2023-05-11 at 5 02 44 PM](https://github.com/afogel/PairApp/assets/2447409/773b0ed0-5fe9-49c2-a4df-98a9a69ca7e0)
